code/modules/mob/living/carbon/inventory.dm

```diff
@@ -34,35 +34,42 @@
 
 	I.screen_loc = null // will get moved if inventory is visible
 	I.loc = src
-	I.equipped(src, slot)
 	I.layer = ABOVE_HUD_LAYER
 
 	switch(slot)

```

Comment:
I'm pretty sure this will result in equipped() never being called when placing an item in any slot other than the ones listed here